Packs/TOPdesk/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### TOPdesk
+- Added fields parameter to get_list_with_query (only return selected fields)
```

Comment:
Please also add the changes made in the command where 'fields' argument was added.

Reply:
I've changed the Release notes.